src/server.rs

```diff
@@ -146,15 +145,41 @@ struct ServerStats {
     /// The count of errors writing to cache.
     pub cache_write_errors: u64,
     /// The number of successful cache writes.
-    pub cache_writes: u32,
+    pub cache_writes: u64,
```

Comment:
Is there a reason you're changing this to `u64`? You're just doing a conversion in the `set_duration_stat` macro below anyway...